admin/src/components/atoms/ExampleStorybook/ExampleStorybook.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ExampleStorybook from './ExampleStorybook';
+import { IExampleStorybook } from './ExampleStorybook.model';
+
+export default {
+  title: 'atoms/ExampleStorybook',
+  component: ExampleStorybook,
+  args: {
+    border: false,
+  },
+  argTypes: {
+    border: {
+      control: {
+        type: 'select',
+        options: [true, false],
+      },
+    },
+  },
+};
+
+export const Default = ({ border }: IExampleStorybook) => <ExampleStorybook border={border} />;
```

Comment:
Miss return type on function

```suggestion
export const Default = ({ border }: IExampleStorybook): JSX.Element => <ExampleStorybook border={border} />;
```